Source/WebKit/UIProcess/ios/WKContentView.mm

```diff
@@ -900,19 +927,17 @@ - (BOOL)_wk_printFormatterRequiresMainThread
     return NO;
 }
 
-- (NSUInteger)_wk_pageCountForPrintFormatter:(_WKWebViewPrintFormatter *)printFormatter
+- (RetainPtr<_WKPrintFormattingAttributes>)_attributesForPrintFormatter:(_WKWebViewPrintFormatter *)printFormatter
 {
     bool isPrintingOnBackgroundThread = !isMainRunLoop();
 
-    [self _waitForDrawToPDFCallbackForPrintFormatterIfNeeded:printFormatter];
```

Comment:
Is it safe to remove this? I see call sites of this method where we no longer call `_waitForDraw...`.